src/server/services/Actions.ts

```diff
@@ -192,6 +211,12 @@ export default class Actions {
     // we got and answer reset the timeout
     if (dripTimeout) clearTimeout(dripTimeout);
 
+    if (isDripSuccessResponse(result)) {
+      await this.updateFaucetBalance().then(() =>
+        logger.info('Updated balance 💰')
```

Comment:
I think you could move this logging to `updateFaucetBalance` directly

Also it should be more specific: `Updated faucet balance`. "Updated balance" doesn't tell me what kind of balance it updated.

Reply:
Didn't want to move the log to actual `updateFaucetBalance` as it happens pretty often and may potentially pollute logs. I'll update wording though, you're right